### ISSUE TITLE
Pull medline data

### ISSUE THREAD

**Issue by mellybelly** [monarch-initiative/dipper#57]
@nlwashington we should figure out if we get this from @eichmann  or pull ourselves. It is updated every day so we probably want a less frequent update cycle? (20-30K new citations per day). It is also very large.

Maybe we only want to pull citations that we already have in the system or that are related to the data set from CTSAsearch.

**Comment by nlwashington** [comment]
how is medline related to pubmed?  same or different?  @ccondit and i have had discussions about this; we need to model these properly (or do you have examples of this?)

**Comment by eichmann** [comment]
MEDLINE is the database curated by NIH-NLM.  PubMed is the public web search engine that uses MEDLINE data (among other things).  Melissa and I were just talking about this.  MEDLINE is ~24 million citations.  There is a daily release of an XML file with additions/deletions/updates that can be upwards of 30k citations.  I load the annual release data (also XML) into a relational database, and then have a cron job (Unix speak for regularly scheduled background process) that processes the daily files every morning around 3 AM.  You have to process the release files in order to maintain proper integrity of the data.

**Comment by mellybelly** [comment]
@tudorgroza can you help us pull in the text mined data from the pub-med browser and get this on a regular cycle? The journals want to use our stuff, but this will make us much more compelling. We could also have an "analyze the literature" search that functions similarly to the PubMed browser does now.

**Comment by drseb** [comment]
@mellybelly please be reminded that we will host the common-disease data at https://github.com/monarch-initiative/hpo-annotation-data , because we will start to manually re-annotated this data. Just in case you would like to use the same data.

**Comment by mellybelly** [comment]
@drseb yes this is exactly what we want to do. will discuss w @tudorgroza, @nicolevasilevsky and I can help curate if we need, just need a plan.

**Comment by drseb** [comment]
Ok. I am just waiting for tudor to send me the latest data (including some pubmed-refs for the annotations), then I will put them in common-diseases folder. Afterwards, we should start re-annotating.
